Comp2025/src/main/java/frc/robot/subsystems/Manipulator.java

```diff
@@ -101,21 +104,19 @@ private enum WristMode
   //      Comp     -177.3  -176.3     -124.7    24.9      25.8
   //      Practice -177.8  -176.8     -124.7    27.3      27.4
   private static final double       kWristAngleRetracted      = Robot.isComp( ) ? -176.3 : -176.8;  // One degree from hardstops
-  private static final double       kWristAngleDeployed       = Robot.isComp( ) ? 24.9 : 27.3;      //
+  // private static final double       kWristAngleDeployed       = Robot.isComp( ) ? 24.9 : 27.3;    Currently being kept for reference
 
-  private static final double       kWristAngleStowed         = 0;
```

Comment:
Ultimately "stowed" may be a different angle than other operations. This implies the wrist has the manipulator in a storage position (like start of match), and we may never use it during the rest of the match. Fully retracted may or may not be the same. What is here is not wrong, but I'm trying to stretch your thinking a little for is likely to occur when we build it up.